File: BasicSample/BasicSample/ViewController.m
//
//  ViewController.m
//  BasicSample
//
//  Created by Michael Han on 12-03-06.
//  Copyright (c) 2012 ForeSee. All rights reserved.
//

#import "ViewController.h"
#import <EXPCore/EXPCore.h>

// must enable modules in your project settings
@import EXPSurveyManagement;

@implementation ViewController

# pragma mark - Actions

- (void)checkEligibility:(id)sender {
  // Show a survey invitation to eligible users
  [EXPSurveyManagement checkIfEligibleForSurvey];
}

- (void)resetState:(id)sender {
  // Reset the state of the SDK (for example after showing an invite, so that
  // the user is eligible to see another one). You wouldn't typically do this
  // in a production app, but it's useful when testing.
  [EXPCore resetState];
}

@end

File: BasicSwiftSample/SwiftSample/ViewController.swift
//
//  ViewController.swift
//  SwiftSample
//
//  Created by Wayne Burkett on 11/7/19.
//  Copyright © 2019 ForeSee. All rights reserved.
//

import UIKit
import EXPCore
import EXPSurveyManagement

class ViewController: UIViewController {
  
  @IBAction func checkEligibilityButtonTouchUpInside(_ sender: Any) {
    // Show a survey invitation to eligible users
    SurveyManagement.checkIfEligibleForSurvey()
  }
  
  @IBAction func resetStateButtonTouchUpInside(_ sender: Any) {
    // Reset the state of the SDK (for example after showing an invite, so that
    // the user is eligible to see another one). You wouldn't typically do this
    // in a production app, but it's useful when testing.
    EXPCore.resetState()
  }
}

